target_postgres/tests/test_target_postgres.py

```diff
@@ -406,11 +445,92 @@ def test_duplicate_records(postgres_target):
     verify_data(postgres_target, "test_duplicate_records", 2, "id", row)
 
 
-def test_array_data(postgres_target):
-    file_name = "array_data.singer"
+def test_array_boolean(postgres_target):
+    file_name = "array_boolean.singer"
+    singer_file_to_target(file_name, postgres_target)
+    row = {"id": 1, "value": [True, False]}
+    verify_data(postgres_target, "array_boolean", 3, "id", row)
+    verify_schema(
+        postgres_target,
+        "array_boolean",
+        check_columns={
+            "id": {"type": BIGINT},
+            "value": {"type": ARRAY},
```

Comment:
I see your point. Let me check how `verify_schema` could be improved accordingly. We are not looking at multiple dimensions here, or eventual nestedness, right? It would just be about verifying the type of the first element within the array against its specification, when the array isn't empty?